src/main/java/org/sunbird/profile/controller/ProfileController.java

```diff
@@ -45,4 +45,10 @@ public ResponseEntity<?> parichayProfileUpdate(@RequestBody Map<String, Object>
 		SBApiResponse response = profileService.userBasicProfileUpdate(request);
 		return new ResponseEntity<>(response, response.getResponseCode());
 	}
+	
+	@PostMapping("/user/v1/signup")
```

Comment:
"/user/v1/signup" , Put this into the constants file and use that variable here. this would ensure the code looks clean and nice

Reply:
why I must pass url pattern na